test/metabase/driver_test.clj

```diff
@@ -64,7 +64,7 @@
                                                               pr-str)))))))
 
 (deftest supports-schemas-matches-describe-database-test
-  (mt/test-drivers (mt/normal-drivers)
```

Comment:
druid is excluded from `mt/normal-drivers`, which is why this test wasn't failing before.